api/src/test/java/org/openmrs/util/OpenmrsUtilTest.java

```diff
@@ -63,6 +72,19 @@
 	        OpenmrsConstants.GLOBAL_PROPERTY_DEFAULT_PATIENT_IDENTIFIER_VALIDATOR,
 	        OpenmrsConstants.LUHN_IDENTIFIER_VALIDATOR);
 	
+	private ConceptService cs;
+	private final String separator = FileSystems.getDefault().getSeparator();
+	
+	@Before
+	public void setup() {
+		cs=Context.getConceptService();
+	}
+	private File createFileAtRelativePath(String relativePath) {
+		String userPath = System.getProperty("user.dir");
+		File file = new File(userPath + separator + relativePath);
+		return file;
```

Comment:
How sure are you that the file is really created? from where do you invoke `File#createNewFile()` ?

Reply:
I create the file in the method of test. This way i can make both directories or file after called this method